Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -718,13 +730,20 @@ extension PaymentSheetTestPlayground {
             "return_url": configuration.returnURL ?? "",
         ] as [String: Any]
 
-        makeRequest(with: PaymentSheetTestPlayground.confirmEndpoint, body: body, completionHandler: { data, _, error in
+        makeRequest(with: PaymentSheetTestPlayground.confirmEndpoint, body: body, completionHandler: { data, response, error in
             guard
                 error == nil,
                 let data = data,
                 let json = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any]
             else {
-                intentCreationCallback(.failure(error ?? ServerSideConfirmationError.unknown))
```

Comment:
Why did we end up showing such a weird error message?  Can we show something a less bad if a merchant doesn't provide a `LocalizedError`? 

maybe we should add some unit tests for https://github.com/stripe/stripe-ios/blob/7791e585d7f08cbdcec029496536c8d97c99d2ab/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Error%2BPaymentSheet.swift#L21

Reply:
I'll work on adding tests, but I want to avoid testing the behavior of `localizedDescription` and rather our computed variable `nonGenericDescription`.